### PR TITLE
parca: minor fixes

### PR DESCRIPTION
**Pull request by Sylfrena**
- profilestore: remove debug value log
  - parcacol: add logging

### CODE REVIEW

File: pkg/parcacol/querier.go [hunk 1]
```diff
@@ -543,6 +548,8 @@ func (q *Querier) QuerySingle(
 func (q *Querier) findSingle(ctx context.Context, query string, t time.Time) (arrow.Record, string, profile.Meta, error) {
 	requestedTime := timestamp.FromTime(t)
 
+	level.Info(q.logger).Log("bamboozling_requested_timestamp", requestedTime, "query", query)
```
Comment (brancz) [inline pkg/parcacol/querier.go:551]: lol .. also I don't think we need this log line anymore

File: pkg/parcacol/ingest.go [hunk 2]
```diff
@@ -114,6 +114,8 @@ func (ing Ingester) IngestProfile(ctx context.Context, ls labels.Labels, p *prof
 	buf.Reset()
 	defer ing.bufferPool.Put(buf)
 
+	level.Info(ing.logger).Log("profile_timestamp", p.Meta.Timestamp)
```
Comment (brancz) [inline pkg/parcacol/ingest.go:117]: no longer needed

**Review by brancz — COMMENTED** [review]
just two small things, I don't think we need the log lines anymore, but I think we should keep the loggers so we can easily add log lines whenever we need to

File: pkg/parcacol/querier.go (hunk 1 above)
Comment (Sylfrena) [reply pkg/parcacol/querier.go:551]: I... thought I had deleted this one. We definitely don't need it. 🙈

**Comment by Sylfrena** [comment]
okay I don't understand why the docs check is failing- precommit passed for me- is there something that we are supposed to run if we change flags?

**Comment by brancz** [comment]
Looks like you fixed it. In case you edited it manually, I think there is make target called `make README.md` that templates the flags into the readme.

Did you maybe forget to push a fix? I can still see both log lines, but the comments are resolved.

**Comment by Sylfrena** [comment]
> Looks like you fixed it. In case you edited it manually, I think there is make target called `make README.md` that templates the flags into the readme.
> 
> Did you maybe forget to push a fix? I can still see both log lines, but the comments are resolved.
I did remove them here https://github.com/parca-dev/parca/pull/2194/commits/17c2eb10164d70b14ce509dcb34191f4cf5038ba
but they came back in the last commit? I don't know what happened there really, I will just repush things

**Review by brancz — APPROVED** [review]

**Pull request merged**